Makefile

```diff
@@ -154,4 +154,12 @@ gen_typescript_types:
 	php artisan typescript:transform
 
 class-leak:
-	vendor/bin/class-leak check app database/migrations config --skip-type Illuminate\\View\\Component
\ No newline at end of file
+	vendor/bin/class-leak check app database/migrations config --skip-type Illuminate\\View\\Component
+
+docker-build-base-image:
+	docker build -t lychee-base:latest ./docker/base
```

Comment:
during each merge to master, a docker image has to be pushed to docker hub. but TBD later. 


right now if I do `make docker-build-dev` and then a `docker-compose up` + changes in the .env file with database value. then a debug version of the Lychee should start up